### ISSUE TITLE
some minor items you may want to add to the installation instructions

### ISSUE THREAD

**Issue by modtastic** [milo2012/osintstalker#11]
geostalker
add pip-2.7 install gdata...otherwise you end up with this error

![gdata](https://f.cloud.github.com/assets/4757954/1389347/74572ef6-3bd5-11e3-8a79-040a5e49b098.jpg)

fbstalker
I believe Facebooks Graph Search is only available for accounts that have set their lanuage to US English.

**Comment by MikeTuffin** [comment]
Facebook Graph search was instantly enabled when switching from English UK to English US.  Verified on two accounts.

**Comment by wujekblade** [comment]
My acc has US English and nothing changes. graph searches still return error that FB doesnt understood my query.

**Comment by MikeTuffin** [comment]
Have you signed up? https://www.facebook.com/about/graphsearch

**Comment by wujekblade** [comment]
@MikeTuffin yes I'm signed up for graph search and I'm using US English, still no luck. Have you got any suggestions left? :)
